hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerConfiguration.java

```diff
@@ -54,7 +54,12 @@
       defaultValue = "10GB", tags = {ConfigTag.BALANCER},
       description = "The maximum size of data in bytes that will be moved " +
           "by Container Balancer.")
-  private long maxSizeToMove = 10 * OzoneConsts.GB;
+  private long maxSizeToMoveInGB = 10 * OzoneConsts.GB;
```

Comment:
We can keep this config same as before. I meant only changing the command line parameter name to maxSizeToMoveInGB. Sorry about the confusion!
Even for the command line I think we will be better off supporting a string representation. But we can do it in a separate PR. +1 o.w.

Reply:
thanks for pointing out this , i will update the parameter.
>Even for the command line I think we will be better off supporting a string representation. But we can do it in a separate PR.

yes, i agree